src/components/NcActionButton/NcActionButton.vue

```diff
@@ -371,7 +371,9 @@ import CheckIcon from 'vue-material-design-icons/Check.vue'
 import ChevronRightIcon from 'vue-material-design-icons/ChevronRight.vue'
 import ChevronLeftIcon from 'vue-material-design-icons/ChevronLeft.vue'
 import ActionTextMixin from '../../mixins/actionText.js'
-import { isRTL } from '@nextcloud/l10n'
+import { isRTL as isRTLFunction } from '@nextcloud/l10n'
+
+const isRTL = isRTLFunction()
```

Comment:
just one line we save here but how about a composable `useIsRtl`?
This could cache it across all components and reduces duplicate logic (even if only one line)

Reply:
Depends on the definition it provides a state (isRTL) and you can inject it in setup context to the component.

It could also be just a module level variable like:
```ts
// utils/rtl.ts
export const isRtl = isRTL()
```

and then you can just use it like
```ts
import { isRtl } from '../../utils/rtl.ts'

// ...
setup() {
    return {
        // ...
        isRtl,
    }
}
// ...
```

Reply:
> ```ts
> import { isRtl } from '../../utils/rtl.ts'
> 
> // ...
> setup() {
>     return {
>         // ...
>         isRtl,
>     }
> }
> // ...
> ```

Agree with this 👍

Wouldn't agree witha composable

```js
const isRtl = l10nIsRtl()

/** Composable to have isRtl */
export function useIsRtl() {
  return isRtl
}

// ----

import { useIsRtl } from '../../composables/rtl.ts'

// ...
setup() {
    return {
        // ...
        isRtl: useIsRtl(),
    }
}
// ...
```

Reply:
Sounds good, let me rearrange files

Reply:
> Agree with this 👍

:+1: